components/admin_console/boolean_setting.tsx

```diff
@@ -64,7 +63,7 @@ export default class BooleanSetting extends React.Component {
                 helpText={helpText}
                 setByEnv={this.props.setByEnv}
             >
-                <a name={this.props.id}/>
+                <a id={this.props.id}/>
```

Comment:
@lieut-data you added this line here, do you know why this line is needed? and why there is a `name` attribute in an `a` label?

Reply:
I believe I added this to enable jumping to these settings from various help texts, e.g. 
```html
This setting cannot be enabled unless the <a href="#Forward80To443">Forward port 80 to 443</a> setting is set to true.
```

For my purposes, `name` and `id` are fungible, but I note that `id` /ought/ to be globally unique in the DOM, whereas `name` only applies to certain HTML elements and is less likely to collide. In practice, I doubt it matters much.

Reply:
From the typescript perspective it matters because is not expected a `name` attribute in the `a` label,  my proposal here would be add some kind of prefix here to avoid collisions. something like `id={'anchor-' + this.props.id}`

Reply:
Why doesn't typescript allow anchors to have a `name` attribute? Isn't this vanilla HTML?

Reply:
Because is not supported in HTML 5, see here https://www.w3schools.com/tags/att_a_name.asp

Reply:
Ahhh, that's the nuance. Thanks for filling in my memory gap :)

For simplicity in this PR, I might suggest just proceeding with `id` and without worrying about the collisions, if only to avoid churn and potential breakage. But I'll defer to your judgement.

Reply:
Sounds good to me :)